### ISSUE TITLE
Offline use case

### ISSUE THREAD

**Issue by keiranmraine** [jimporter/mike#55]
Hi,

I'm using mkdocs material in a slightly different way, gradually expanding data in a site.  We retain all the md, just add more and allow mkdocs to rebuild the whole thing each time.

Obviously over time this will become prohibitive so the idea of being able to push monthly blocks into the version style is very attractive.

Can `mike` do this, or do you have any plans for an offline model?  Are you aware of any other tools that can do this?

Basically looking to work on folders locally rather than branches or tags:

edit: wrong key combination submitted above without example, sorry

```
./latest/md -> versions/2021.04/md
./versions/2021.01/md
./versions/2021.02/md
./versions/2021.03/md
./versions/2021.04/md
...
```

**Comment by jimporter** [comment]
The vast majority of mike's code is for working with git branches, so if that's not what you want, I don't think mike could be adapted for your use case. While you *can* use it locally by just checking out the branch, that doesn't sound like what you want.

It sounds like all you really need is to change the `site_dir` field in `mkdocs.yml` (or on the command line) every month. There's probably even a programmatic way to do this. Then you wouldn't need any other tools like mike.

I'm going to close this since I'm 99% sure mike won't be able to fit your needs, but feel free to comment further if you want to continue the discussion here.
